bigquery/google/cloud/bigquery/schema.py

```diff
@@ -14,7 +14,7 @@
 
 """Schemas for BigQuery tables / queries."""
 
-import collections
+from collections.abc import Mapping
```

Comment:
import modules not classes (I know we're not consistent about this)

```suggestion
from collections import abc
```